### PR TITLE
add command to nullify votes by one user on a given target user, optionally within a date range

### PR DESCRIPTION
**Pull request by b0b3rt**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1205752948760379) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/server/ckEditor/ckEditorWebhook.ts [hunk 1]
```diff
@@ -376,12 +376,12 @@ async function fetchCkEditorRestAPI(method: string, uri: string, body?: any): Pr
 Globals.fetchCkEditorRestAPI = fetchCkEditorRestAPI;
 
 async function flushCkEditorCollaboration(ckEditorId: string) {
-  await fetchCkEditorRestAPI("DELETE", `/collaborations/${ckEditorId}`);
+  await fetchCkEditorRestAPI("DELETE", `/collaborations/${ckEditorId}?force=true&wait=true`);
```
Comment (Discordius) [inline packages/lesswrong/server/ckEditor/ckEditorWebhook.ts:379]: This probably shouldn't be part of this PR

File: packages/lesswrong/server/scripts/nullifyVotes.ts [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
+  let beforeDate = undefined;
+
+  if (dateRange.after) {
+    afterDate = new Date(dateRange.after);
+    if (isNaN(afterDate.getTime())) throw new Error('Invalid after date provided!');
+  }
+
+  if (dateRange.before) {
+    beforeDate = new Date(dateRange.before);
+    if (isNaN(beforeDate.getTime())) throw new Error('Invalid after date provided!');
```
Comment (Discordius) [inline packages/lesswrong/server/scripts/nullifyVotes.ts:30]: Wrong error message

File: packages/lesswrong/server/scripts/nullifyVotes.ts [hunk 3]
```diff
@@ -1,12 +1,36 @@
...
+Globals.nullifyVotesForNullifiedUsers = async () => {
   const users = await Users.find({nullifyVotes: true}).fetch();
   users.forEach((user) => {
     void nullifyVotesForUser(user);
   })
   //eslint-disable-next-line no-console
   console.warn(`Nullified votes for ${users.length} users`);
 }
+
+interface DateRange {
```
Comment (oetherington) [inline packages/lesswrong/server/scripts/nullifyVotes.ts:14]: I think it's a a little confusing that these two files each declare an interface with the same name but incompatible types. Probably one should be renamed (`UserDateRange` or `DateRangeInput` maybe?), or the global function could just be `(sourceUserId: string, targetUserId: string, after?: string, before?: string)`

**Review by oetherington — APPROVED** [review]
Nice - I've definitely wanted this at some point in the past!

File: packages/lesswrong/server/scripts/nullifyVotes.ts (hunk 3 above)
Comment (b0b3rt) [reply packages/lesswrong/server/scripts/nullifyVotes.ts:14]: Yeah, `DateRangeInput` seems good.
